resources/lang/de.json

```diff
@@ -77,27 +77,28 @@
     "user.fresh-link": "Ein neuer Bestätigungslink wurde an Deine E-Mail-Adresse geschickt.",
     "user.header-reset-pw": "Passwort zurücksetzen",
     "user.login": "Anmelden",
-    "user.mastodon-instance-url": "Instanz",
-    "user.not-received-before": "Wenn Du die E-Mail nicht bekommen hast, ",
-    "user.not-received-link": "klicke hier für eine neue E-Mail",
-    "user.password": "Passwort",
-    "user.please-check": "Bevor Du weitergehst, schau in Deine Emails für einen Bestätigungslink.",
-    "user.register": "Registrieren",
-    "user.remember-me": "Angemeldet bleiben",
-    "user.reset-pw-link": "Wiederherstellungs-Link schicken",
-    "user.username": "Nutzername",
-    "user.you": "Du",
-    "user.home-set": "Deine Heimat ist nun :Station.",
-    "user.always-dbl": "Mit #dbl twittern?",
-    "user.private-profile": "Privates Profil",
-    "user.liked-status": "gefällt dieser Status.",
-    "user.liked-own-status": "gefällt der eigene Status.",
-    "user.invalid-mastodon": ":domain scheint keine Mastodon-Instanz zu sein.",
-    "user.no-user": "Keinen Nutzer gefunden.",
-    "dates.-on-": " um ",
-    "dates.decimal_point": ",",
-    "dates.thousands_sep": ".",
-    "dates.April": "April",
+  "user.mastodon-instance-url": "Instanz",
+  "user.not-received-before": "Wenn Du die E-Mail nicht bekommen hast, ",
+  "user.not-received-link": "klicke hier für eine neue E-Mail",
+  "user.password": "Passwort",
+  "user.please-check": "Bevor Du weitergehst, schau in Deine Emails für einen Bestätigungslink.",
+  "user.register": "Registrieren",
+  "user.remember-me": "Angemeldet bleiben",
+  "user.reset-pw-link": "Wiederherstellungs-Link schicken",
+  "user.username": "Nutzername",
+  "user.you": "Du",
+  "user.home-set": "Dein Heimatbahnhof ist nun :Station.",
+  "user.home-not-set": "Du hast noch keinen Heimatbahnhof festgelegt.",
+  "user.always-dbl": "Mit #dbl twittern?",
+  "user.private-profile": "Privates Profil",
+  "user.liked-status": "gefällt dieser Status.",
+  "user.liked-own-status": "gefällt der eigene Status.",
+  "user.invalid-mastodon": ":domain scheint keine Mastodon-Instanz zu sein.",
+  "user.no-user": "Keinen Nutzer gefunden.",
+  "dates.-on-": " um ",
+  "dates.decimal_point": ",",
+  "dates.thousands_sep": ".",
+  "dates.April": "April",
```

Comment:
What happened here? This would cause conflicts with weblate.

Reply:
Maybe phpstorm reformat? does this really cause conflicts and what should i do to fix this? Just indent it again?

Reply:
Yes, just intend. You can enable the .editorconfig to prevent phpstorm to format it like these. It's disabled by default.